testbench/test-exec/packages/test-package/test-version/set-cache.py

```diff
@@ -16,4 +16,7 @@
 print("foo = bar >> VELOCITAS_CACHE")
 print("x=y >>  VELOCITAS_CACHE")
 print("0123='random' >> VELOCITAS_CACHE")
+print('arr1=["/path/test1", "/path/test2"] >> VELOCITAS_CACHE')
+print('arr2=["/path/test1" ,"/path/test2"] >> VELOCITAS_CACHE')
```

Comment:
it seems that the system tests are not parsing this arr2 correctly. please check why